### PR TITLE
Phrase updated. Arduino Opta removed from subtitle, Tech table update…

### PR DESCRIPTION
**Pull request by jcarolinares**
…d to fit Trademarks and programming languages

## What This PR Changes
- Product page updated to fix some small inconsistencies

## Contribution Guidelines
- [ x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Review by aliphys — APPROVED** [review]
LGTM

**Review by jhansson-ard — APPROVED** [review]
LGTM!

File: content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml [hunk 1]
```diff
@@ -38,4 +38,4 @@ RTC: ~10days, NTP sync through ethernet
 Secure element: ATECC608B
 Programming:
   Arduino programming language: Via Arduino IDEs, Arduino CLI, Arduino Web Editor
-  IEC-61131-3 as option: Ladder Diagram (LD), Function Block Diagram (FBD), Sequential Function Chart (SFC), Structured Text (ST), Instruction List (IL)
+  IEC-61131-3: Ladder Diagram (LD), Function Block Diagram (FBD), Sequential Function Chart (SFC), Structured Text (ST), Instruction List (IL)
```
Comment (marqdevx) [inline content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml:41]: Add "Through PLC IDE" like the Arduino programming 🤔

Comment (marqdevx) [inline content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml:41]:
```suggestion
  IEC-61131-3 (Only via  Arduino PLC IDE): Ladder Diagram (LD), Function Block Diagram (FBD), Sequential Function Chart (SFC), Structured Text (ST), Instruction List (IL)
```

**Comment by sebromero** [comment]
@jcarolinares Can you have a look please?

**Comment by marqdevx** [comment]
Let me know if you need help with the conflicts! 👁️👃👁️

**Pull request closed without merging**